test/functional/tests/io_class/test_io_class_purge.py

```diff
@@ -0,0 +1,148 @@
+#
+# Copyright(c) 2020 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+from collections import namedtuple
+from datetime import timedelta
+from math import floor
+import random
+
+import pytest
+
+from .io_class_common import *
+from api.cas.casadm_params import OutputFormat
+from api.cas.ioclass_config import IoClass
+from api.cas.statistics import UsageStats
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+from test_tools import fs_utils
+from test_tools.dd import Dd
+from test_tools.disk_utils import Filesystem
+from test_tools.fio.fio import Fio
+from test_tools.fio.fio_param import ReadWrite, IoEngine
+from test_utils.filesystem.file import File
+from test_utils.os_utils import drop_caches, DropCachesMode, sync, Udev
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_ioclass_usage_sum():
+    """
+        title: Test for ioclass stats after purge
+        description: |
+          Create ioclasses for 3 different directories. Run IO against each
+          directory, check usage stats correctness before and after purge
+        pass_criteria:
+          - Usage stats are consistent on each test step
+          - Usage stats doesn't exceed cache size
+    """
+    with TestRun.step("Prepare disks"):
+        cache, core = prepare()
+        cache_size = cache.get_statistics().config_stats.cache_size
+
+    with TestRun.step("Disable udev"):
+        Udev.disable()
+
+    with TestRun.step(f"Preparing filesystem and mounting {core.system_path} at {mountpoint}"):
```

Comment:
Please use the same verbs form in each test step description, e.g. prepare, add, etc.